epaint/src/stats.rs

```diff
@@ -211,6 +211,13 @@ impl PaintStats {
             Shape::Mesh(mesh) => {
                 self.shape_mesh += AllocInfo::from_mesh(mesh);
             }
+            Shape::CubicBezier(_) => {
+                // todo!("CubicBezier");
+                // panic!("CubicBezier");
+            }
+            Shape::QuadraticBezier(_) => {
+                // todo!("QuadraticBezier");
+            }
```

Comment:
They can be put in the same match arm as `Shape::Circle`, `Shape::LineSegment` etc. These stats are about internal allocations, and these don't have any dynamic allocations in them

Reply:
yes, moved them together.